### PR TITLE
Change LB config files to use same binning

### PR DESCRIPTION
**Pull request by ausonandres**
The purpose of this PR is to change Low Background configuration files to build always 50x50 maps. To do that, ``nmin`` parameter is modified. Besides that, chi2 range has also been changed according to new definition.

### CODE REVIEW

**Comment by bpalmeiro** [comment]
What Alberto meant, and will change accordingly is a 50x50 map

**Comment by ausonandres** [comment]
Sorry, it's 50x50 map, indeed.

**Review by bpalmeiro — COMMENTED** [review]
Looks good to me. Just wondering if we should implement a system as CERES for the config file

File: krcal/map_builder/map_builder_functions_test.py [hunk 1]
```diff
@@ -41,17 +41,22 @@ def test_scrip_runs_and_produces_correct_outputs(folder_test_dst  ,
...
-                       run_number     = run_number     ))
+                       run_number     = run_number     ,
+                       map_params     = map_params_new ))
     map_builder(config.as_namespace)
     maps = read_maps(map_file_out)
     assert type(maps)==ASectorMap
 
     old_maps = read_maps(test_map_file)
+    print(old_maps.e0)
+    print(maps.e0)
```
Comment (bpalmeiro) [inline krcal/map_builder/map_builder_functions_test.py]: may you remove those prints?

Comment (ausonandres) [reply krcal/map_builder/map_builder_functions_test.py]: Ah sorry, my fault. I've removed them.

**Review by bpalmeiro — APPROVED** [review]
As nobody complained, and this is a little but needed change, I approve it. Nevertheless, I will suggest the merging waits till the big PR #30 is merged.

Good job!

**Pull request merged**